test/e2e/testing-manifests/auth/encrypt/connect-registry.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Let's move the logic in this script to a function in `test/e2e/testing-manifests/auth/encrypt/setup-cluster.sh`